Docs/User Guide/User Guide.md

```diff
@@ -10,6 +10,10 @@ A valid hardware configuration for the core must be provided to each instance of
 
 ![BTI XML Editor](Screenshots/BTI_XML_Configuration.PNG)
 
+Additionally, the `name` attribute for each label in the hardware configuration file should adhere to the following format in order to be recognized by the custom device: name="label #" where # is the `labelDecimal`, e.g. "label 16".  If the label uses SDI, the label should also include `_N` at the end of the label, e.g. "label 16_1" for SDI=01, or "label 16_3" for SDI=11.  The `BTI XML Editor` will generate these label names by default.
```

Comment:
The `BTI XML Editor` will _not_ generate these label names by default. VeriStand generates the channel names using that convention, so you'll have to update the hardware XML.

Reply:
updated in commit 3